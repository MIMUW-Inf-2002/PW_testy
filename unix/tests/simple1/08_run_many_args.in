run ls 08_run_many_args.out
sleep 100
out 0
run echo secret message here
sleep 100
out 1
