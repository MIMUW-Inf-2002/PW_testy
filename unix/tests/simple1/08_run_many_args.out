Task 0 started: pid 62209.
Task 0 ended: status 0.
Task 0 stdout: '08_run_many_args.out'.
Task 1 started: pid 62213.
Task 1 ended: status 0.
Task 1 stdout: 'secret message here'.
